tests/pytests/test_dawn_drivers.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+import ale
+from ale.drivers import dawn_drivers
+from ale.base import data_naif
+from ale.base import label_pds3
+
+from unittest.mock import PropertyMock, patch
+
+# 'Mock' the spice module where it is imported
+from conftest import SimpleSpice, get_mockkernels
+
+simplespice = SimpleSpice()
+
+data_naif.spice = simplespice
+dawn_drivers.spice = simplespice
+label_pds3.spice = simplespice
+
+from ale.drivers.dawn_drivers import DawnFcPds3NaifSpiceDriver
+
+DawnFcPds3NaifSpiceDriver.metakernel = get_mockkernels
+
+@pytest.fixture
+def driver():
+    return DawnFcPds3NaifSpiceDriver("")
+
+def test_instrument_id(driver):
+    with patch('ale.base.label_pds3.Pds3Label.instrument_id', new_callable=PropertyMock) as mock_instrument_id:
+        mock_instrument_id.return_value = 1
+        with patch('ale.base.label_pds3.Pds3Label.filter_number', new_callable=PropertyMock) as mock_filter_number:
+            mock_filter_number.return_value = 2
+            assert driver.instrument_id == 'DAWN_1_FILTER_2'
```

Comment:
A possibly cleaner solution to patch so we will not have to deal with nested ```with``` blocks:
```
@patch('ale.base.label_pds3.Pds3Label.instrument_id', 1)
@patch('ale.base.label_pds3.Pds3Label.filter_number', 2)
def test_instrument_id(driver):
    assert driver.instrument_id == 'DAWN_1_FILTER_2
```

Reply:
(I tested that this works in your test code as desired.)

Reply:
@SgStapleton Thanks for testing this. I think that this is a lot cleaner to read. Looks great! @paarongiroux Could you update these to this format? We should also get the testing docs updated for down stream people writing drivers.